packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -383,6 +395,12 @@ class _DragTargetState<T> extends State<DragTarget<T>> {
   final List<_DragAvatar<T>> _candidateAvatars = <_DragAvatar<T>>[];
   final List<_DragAvatar<dynamic>> _rejectedAvatars = <_DragAvatar<dynamic>>[];
 
+  void onDrag(_DragAvatar<dynamic> avatar, Offset lastOffset) {
```

Comment:
Ah, I see the issue. You're calling this for all drag targets, not just those that have been accepted. Was this your intent?